app/messaging/application/index.js

```diff
@@ -8,8 +8,6 @@ async function getApplication (applicationReference, sessionId) {
 }
 
 async function sendApplication (application, sessionId) {
-  console.log(`Sending application ${JSON.stringify(application)} to queue ${applicationRequestQueue.address} with sessionID ${sessionId}.`)
```

Comment:
Maybe worth keeping this line but via proper logger

Reply:
Don't think we want to be logging the sessionId or the users application (as it contains personal details). What I've done here is put the temporary application id onto the logger before we make this call, and the new one afterwards if successful. https://github.com/DEFRA/ffc-ahwr-farmer-apply/pull/424/files#diff-678633002b18fd55567e0abbbfbbb8f6dd06d19f3824dfb27e88425de276d29dR50-R83

So if this errors you'll get something like:
```
{
  "level": "error",
  "time": 1729080472899,
  "pid": 95,
  "hostname": "02708ebe0a24",
  "name": "ffc-ahwr-farmer-apply",
  "req": {
    "id": "1729080471237:02708ebe0a24:95:m2bttint:10028",
    "method": "post",
    "url": "/apply/declaration"
  },
  "tempApplicationReference": "TEMP-4251-9621",
  "sbi": "107167406",
  "statusCode": 500,
  "message": "An internal server error occurred",
  "stack": "Error: Apply declaration returned a null application reference.
    at handler (/home/node/app/routes/declaration.js:89:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async exports.Manager.execute (/home/node/node_modules/@hapi/hapi/lib/toolkit.js:60:28)
    at async internals.handler (/home/node/node_modules/@hapi/hapi/lib/handler.js:46:20)
    at async exports.execute (/home/node/node_modules/@hapi/hapi/lib/handler.js:31:20)
    at async Request._lifecycle (/home/node/node_modules/@hapi/hapi/lib/request.js:371:32)
    at async Request._execute (/home/node/node_modules/@hapi/hapi/lib/request.js:281:9)",
  "msg": "pre response error"
}
```

And if it succeeds you'll see something like:

```
{
  "level": "info",
  "time": 1729083940385,
  "pid": 303,
  "hostname": "02708ebe0a24",
  "name": "ffc-ahwr-farmer-apply",
  "req": {
    "id": "1729083938284:02708ebe0a24:303:m2bvvts4:10061",
    "method": "post",
    "url": "/apply/endemics/declaration"
  },
  "tempApplicationReference": "TEMP-72B2-1616",
  "sbi": "107167406",
  "newApplicationReference": "IAHW-63B1-8D68",
  "res": { "statusCode": 200 },
  "responseTime": 2101,
  "msg": "[response] post /apply/endemics/declaration 200 (2101ms)"
}
```

Reply:
Definitely would not want the pii bits logged out, but didn;t know if a log line saying sent to queue provided value. To be fair thinking about it, it probably doesn;t add anything